src/client/datascience/interactive-common/notebookProvider.ts

```diff
@@ -43,26 +44,27 @@ export class NotebookProvider implements INotebookProvider {
             }
         });
         // Connect to either a jupyter server or a stubbed out raw notebook "connection"
-        if (this.rawNotebookProvider.isSupported) {
+        if (this.rawNotebookProvider.isSupported && !options.local) {
```

Comment:
This seems backwards? Raw is always local?

Reply:
Fixed the property, it was poorly named, renamed to `localJupyter` that propery would be true ONLY when starting jupyter for local